ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/CreateVulnReport/CreateVulnReportPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { PageSection, Title, Divider, Flex, FlexItem } from '@patternfly/react-core';
+
+import PageTitle from 'Components/PageTitle';
+
+function VulnReportsPage() {
+    return (
+        <>
+            <PageTitle title="Vulnerability reporting - create" />
+            <Divider component="div" />
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Flex direction={{ default: 'column' }} className="pf-u-py-lg pf-u-px-lg">
+                    <FlexItem>
+                        <Title headingLevel="h1">Create report</Title>
+                    </FlexItem>
+                    <FlexItem>
+                        Configure reports, define report scopes, and assign distribution lists to
+                        report on vulnerabilities across the organization.
+                    </FlexItem>
+                </Flex>
+            </PageSection>
+            <PageSection padding={{ default: 'noPadding' }}>
+                <PageSection isCenterAligned />
+            </PageSection>
```

Comment:
I noticed us doing this in other places, but I think the nested <PageSection> components are redundant. I think if you leave off the padding=noPadding part, you'd get the same effect.